### PR TITLE
[NOD-675] Replace start-hash/stop-hash with meaningful names

### PR DESCRIPTION
**Pull request by dereeno**

### CODE REVIEW

File: rpcclient/dag.go [hunk 1]
```diff
@@ -256,7 +256,7 @@ func (c *Client) GetBlockCount() (int64, error) {
 type FutureGetChainFromBlockResult chan *response
 
 // Receive waits for the response promised by the future and returns the selected
-// parent chain starting from startHash up to the virtual. If startHash is not in
+// parent chain starting from lowHash up to the virtual. If lowHash is not in
```
Comment (someone235) [inline rpcclient/dag.go]: Please keep the name `startHash` in `getChainFromBlock`

File: rpcmodel/rpc_commands.go [hunk 2]
```diff
@@ -293,15 +293,15 @@ func NewGetBlockTemplateCmd(request *TemplateRequest) *GetBlockTemplateCmd {
 // GetChainFromBlockCmd defines the getChainFromBlock JSON-RPC command.
 type GetChainFromBlockCmd struct {
 	IncludeBlocks bool    `json:"includeBlocks"`
-	StartHash     *string `json:"startHash"`
+	LowHash       *string `json:"lowHash"`
```
Comment (someone235) [inline rpcmodel/rpc_commands.go]: fix to startHash

File: rpcmodel/rpc_commands.go [hunk 3]
```diff
@@ -293,15 +293,15 @@ func NewGetBlockTemplateCmd(request *TemplateRequest) *GetBlockTemplateCmd {
...
 }
 
 // NewGetChainFromBlockCmd returns a new instance which can be used to issue a
 // GetChainFromBlock JSON-RPC command.
-func NewGetChainFromBlockCmd(includeBlocks bool, startHash *string) *GetChainFromBlockCmd {
+func NewGetChainFromBlockCmd(includeBlocks bool, lowHash *string) *GetChainFromBlockCmd {
```
Comment (someone235) [inline rpcmodel/rpc_commands.go]: fix to startHash

File: rpcmodel/rpc_commands.go [hunk 4]
```diff
@@ -293,15 +293,15 @@ func NewGetBlockTemplateCmd(request *TemplateRequest) *GetBlockTemplateCmd {
...
 	return &GetChainFromBlockCmd{
 		IncludeBlocks: includeBlocks,
-		StartHash:     startHash,
+		LowHash:       lowHash,
```
Comment (someone235) [inline rpcmodel/rpc_commands.go]: fix to startHash

File: rpcmodel/rpc_commands_test.go [hunk 5]
```diff
@@ -987,7 +987,7 @@ func TestRPCServerCommands(t *testing.T) {
 			unmarshalled: &rpcmodel.GetTopHeadersCmd{},
 		},
 		{
-			name: "getTopHeaders - with start hash",
+			name: "getTopHeaders - with low hash",
```
Comment (someone235) [inline rpcmodel/rpc_commands_test.go]: fix to startHash

File: server/rpc/handle_get_chain_from_block.go [hunk 6]
```diff
@@ -24,29 +24,29 @@ func handleGetChainFromBlock(s *Server, cmd interface{}, closeChan <-chan struct
 	}
 
 	c := cmd.(*rpcmodel.GetChainFromBlockCmd)
-	var startHash *daghash.Hash
-	if c.StartHash != nil {
-		startHash = &daghash.Hash{}
-		err := daghash.Decode(startHash, *c.StartHash)
+	var lowHash *daghash.Hash
```
Comment (someone235) [inline server/rpc/handle_get_chain_from_block.go]: fix to startHash

File: server/rpc/rpcserverhelp.go [hunk 7]
```diff
@@ -172,14 +172,14 @@ var helpDescsEnUS = map[string]string{
...
 	"getBlocks-includeRawBlockData":     "If set to true - the raw block data would be also included.",
 	"getBlocks-includeVerboseBlockData": "If set to true - the verbose block data would also be included.",
-	"getBlocks-startHash":               "Hash of the block with the bottom blue score. If this hash is unknown - returns an error.",
-	"getBlocks--result0":                "Blocks starting from startHash. The result may contains up to 1000 blocks. For the remainder, call the command again with the bluest block's hash.",
+	"getBlocks-lowHash":                 "Hash of the block with the bottom blue score. If this hash is unknown - returns an error.",
+	"getBlocks--result0":                "Blocks starting from lowHash. The result may contains up to 1000 blocks. For the remainder, call the command again with the bluest block's hash.",
 
 	// GetChainFromBlockResult help.
-	"getBlocksResult-hashes":        "List of hashes from StartHash (excluding StartHash) ordered by smallest blue score to greatest.",
+	"getBlocksResult-hashes":        "List of hashes from LowHash (excluding LowHash) ordered by smallest blue score to greatest.",
```
Comment (someone235) [inline server/rpc/rpcserverhelp.go]: fix to startHash

File: server/rpc/rpcserverhelp.go [hunk 8]
```diff
@@ -357,14 +357,14 @@ var helpDescsEnUS = map[string]string{
 	"getBlockTemplate--result1":    "An error string which represents why the proposal was rejected or nothing if accepted",
 
 	// GetChainFromBlockCmd help.
-	"getChainFromBlock--synopsis":     "Return the selected parent chain starting from startHash up to the virtual. If startHash is not in the selected parent chain, it goes down the DAG until it does reach a hash in the selected parent chain while collecting hashes into removedChainBlockHashes.",
-	"getChainFromBlock-startHash":     "Hash of the bottom of the requested chain. If this hash is unknown or is not a chain block - returns an error.",
+	"getChainFromBlock--synopsis":     "Return the selected parent chain starting from lowHash up to the virtual. If lowHash is not in the selected parent chain, it goes down the DAG until it does reach a hash in the selected parent chain while collecting hashes into removedChainBlockHashes.",
```
Comment (someone235) [inline server/rpc/rpcserverhelp.go]: fix to startHash

File: server/rpc/rpcserverhelp.go [hunk 9]
```diff
@@ -357,14 +357,14 @@ var helpDescsEnUS = map[string]string{
...
+	"getChainFromBlock-lowHash":       "Hash of the bottom of the requested chain. If this hash is unknown or is not a chain block - returns an error.",
```
Comment (someone235) [inline server/rpc/rpcserverhelp.go]: fix to startHash

File: server/rpc/rpcserverhelp.go [hunk 10]
```diff
@@ -357,14 +357,14 @@ var helpDescsEnUS = map[string]string{
...
 	"getChainFromBlock-includeBlocks": "If set to true - the block contents would be also included.",
 	"getChainFromBlock--result0":      "The selected parent chain. The result may contains up to 1000 blocks. For the remainder, call the command again with the bluest block's hash.",
 
 	// GetChainFromBlockResult help.
 	"getChainFromBlockResult-removedChainBlockHashes": "List chain-block hashes that were removed from the selected parent chain in top-to-bottom order",
-	"getChainFromBlockResult-addedChainBlocks":        "List of ChainBlocks from Virtual.SelectedTipHashAndBlueScore to StartHash (excluding StartHash) ordered bottom-to-top.",
+	"getChainFromBlockResult-addedChainBlocks":        "List of ChainBlocks from Virtual.SelectedTipHashAndBlueScore to LowHash (excluding LowHash) ordered bottom-to-top.",
```
Comment (someone235) [inline server/rpc/rpcserverhelp.go]: fix to startHash

File: rpcmodel/rpc_commands_test.go (hunk 5 above)
Comment (dereeno) [reply rpcmodel/rpc_commands_test.go]: I believe this is a mistake

**Pull request merged**